src/lib/invoiceParser.ts

```
@@ -0,0 +1,211 @@
+      throw new InvoiceParserError(`Amount for "${label}" not found`)
+    }
+    return amount
+  }
+
+  private static parseAmountToMilliDollars(amountText: string): number {
+    const cleanText = amountText.replace(/[$,]/g, '')
+    const formatRegex = /^-?\d+\.\d{2}$/
+    if (!formatRegex.test(cleanText)) {
+      throw new InvoiceParserError(`Amount "${amountText}" does not match required format (must end with .XX)`)
+    }
+    const dollarAmount = parseFloat(cleanText)
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)